configure.ac

```diff
@@ -857,15 +857,16 @@ AS_IF([test -n "$LOKIT_PATH"],
       [CPPFLAGS="$CPPFLAGS -I${LOKIT_PATH}"])
 lokit_msg="$LOKIT_PATH"
 
-AS_IF([test "$ENABLE_IOSAPP" != "true" -a "$ENABLE_ANDROIDAPP" != "true" -a "$enable_fuzzers" != "yes"],
+AS_IF([test "$ENABLE_IOSAPP" != "true" -a "$ENABLE_ANDROIDAPP" != "true"],
```

Comment:
I didn't need this as I always configure a git checkout for fuzzing or not and never switch between the two, but why not.

Reply:
Right, switching is how I hit it and thought it's not a bad idea to be permissive.